### ISSUE TITLE
Docker build doesn't respect `Gemfile.lock`

### ISSUE THREAD

**Issue by jagthedrummer** [navarasu/serverless-ruby-layer#40]
**Describe the bug**

My project has a `Gemfile` and a `Gemfile.lock` but when I `sls deploy` I end up with a different set of gems being installed than what is specified in the lock file. If I remove `Gemfile.lock` the build process also does not generate one, which it seems like it should.

**Error Stack Trace**

None

**Example Config files**

My `Gemfile.lock` contains:
```
aws-sdk-core (3.113.0)
```

But the `.serverless/ruby_layer/gemLayer.zip` package contains version `3.113.1`.
```
unzip -l .serverless/ruby_layer/gemLayer.zip | grep aws-sdk-core
   0  04-02-2021 18:17   ruby/2.7.0/gems/aws-sdk-core-3.113.1/
...
```

**Desktop (please complete the following information):**
 - OS: [e.g. Mac OS]
 - NodeVersion [e.g. 12.16.1]
 - Ruby version [e.g. 2.5.8]
 - Bundler version [e.g. 2.1.2]

**Additional context**

**Comment by navarasu** [comment]
After merging your valuable changes, I have created a proper gemfile.lock example and test case for both basic and docker case.
* Basic case without docker works without issue.
* With docker case, we need to take care of the missing bundler version else it will fail with the following error
```
 Could not find 'bundler' (1.16.6) required by your /var/gem_build/Gemfile.lock. (Gem::GemNotFoundException)
To update to the latest version installed on your system, run `bundle update --bundler`.
To install the missing version, run `gem install bundler:1.16.6`
```
Currently working on handling this version missing case.

**Comment by navarasu** [comment]
Handled missing bundler version error for use_docker case by installing respecting bundler. Now the test case is green.

**Comment by navarasu** [comment]
@jagthedrummer  This fix is included in release 1.5.0. Please update the npm version and verify.
```
npm i serverless-ruby-layer
```
